badge/badge.css

```diff
@@ -0,0 +1,57 @@
+@import '../style.css';
+
+.badge-container{
+    /* display: flex; */
+    display: inline-block;
+    /* flex-direction: column; */
+    /* background-color: blueviolet; */
+    /* flex-basis: content; */
+    width: auto;
+    position: relative;
+    padding: 0.2rem;
+
+}
+
+.badge{
+    padding: 0.2rem;
+    padding-right: 10px;
+    border-radius: 15px 50px 30px;
+    color: #ffffff !important;
```

Comment:
You should not use !important here. It is to be used is very rare cases.

Reply:
removed important